src/pages/TeamPage/TeamPage.tsx

```diff
@@ -220,4 +259,31 @@ export class TeamPage extends React.Component<ITeamPageProps, ITeamPageState> {
     });
   };
 
+  private __handleChange = (id: String) => {
+      switch(id) {
```

Comment:
This could be simplified using a map, which is preferred over `switch`.
```
private __handleChange = () => {
      const id2Hours = {
        'hour1': 1,
        'hour2': 2,
        'hour4': 4,
        'hour8': 8,
        'hour16': 16
      };
  
      this.setState({forecastingHours: id2Hours[id]});
    };
```